calculator.rb

```diff
@@ -0,0 +1,139 @@
+# ROOM FOR IMPROVEMENT
+# Support parentheticals
+# Support commas in longer numbers
+# Allow for equation with more than 2 numbers
+# Support square root
+# Ask whether to end or continue after outputting result
+
+# -------------
+
+# CHECKS IF USER INPUT IS A VALID OPERATION
+def is_operator?(user_input)
+  operators = ["+", "add", "addition", "plus",
+               "-", "sub", "subtract", "subtraction",
+               "*", "multiply", "times",
+               "/", "divide", "division",
+               "^", "**", "power", "exponent", "exponents", "raise",
+               "%", "modulo", "modulus", "remainder"
+  ]
+
+  if operators.include?(user_input)
```

Comment:
Think about how the `while` in your `else` statement works. Do you need this `if/else` block at all, or can you see a way of just having the `while` loop?